.github/workflows/tour_of_beam_backend.yml

```diff
@@ -28,6 +28,11 @@ on:
     tags: 'v*'
     paths: ['learning/tour-of-beam/backend/**']
 
+# This allows a subsequently queued workflow run to interrupt previous runs
+concurrency:
+  group: '${{ github.workflow }} @ ${{ github.event.pull_request.head.label || github.head_ref || github.ref }}'
+  cancel-in-progress: true
```

Comment:
FYI, I added https://github.com/apache/beam/issues/23089 because I think we should be doing this more broadly - thanks for introducing it to the repo 😄